src/zzz_od/application/hollow_zero/lost_void/operation/lost_void_run_level.py

```
@@ -696,122 +697,115 @@ def in_battle(self) -> OperationRoundResult:
+                        frame_result: DetectFrameResult = gpu_executor.execute_function(
+                            self.ctx.model_config.lost_void_det_gpu,
+                            self.detector.run,
+                            image=self.last_screenshot,
+                            conf=0.9,
+                            run_time=self.last_screenshot_time,
+                        )
                         with_interact, with_distance, with_entry = self.detector.is_frame_with_all(frame_result)
                         if with_interact or with_distance or with_entry:
-                            no_in_battle = True
+                            not_in_battle = True
                     except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)